opp-distribution-service/contracts/testnet/sepolia-to-boba/scripts/deploy.js

```diff
@@ -0,0 +1,19 @@
+const { ethers } = require("hardhat");
+
+async function main() {
+    // get the contract factory
+    const BridgeContract = await ethers.getContractFactory("SepoliaToBobaBridge");
+
+
+    // deploy contract
+    const bridgeContract = await BridgeContract.deploy();
+    await bridgeContract.deployed();
+    console.log("SepoliaToBobaBridge deployed to: ", bridgeContract.address);
```

Comment:
Need to keep track of this, since a new deploy will create a new address

Reply:
If funds exist, create contract to move them over: https://sepolia.etherscan.io/address/0x7fe4c46F6176a744a2c3784306a14953902122A0